notebooks/abbash_notebook.ipynb

```diff
@@ -1,28 +1,540 @@
 {
```

Comment:
<p> </p><ol><li>create a sine-wave and plot</li><li>plot time-series data</li><li>generate FFT on sine-wave data</li><li>plot amplitude vs frequency from FFT</li><li>use <code>np.loadtxt</code> to   import experimental data</li><li>Find natural frequency of a vibrating beam</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>